### ISSUE TITLE
At startup on small windows, legend is always shown

### ISSUE THREAD

**Issue by pcav** [3liz/lizmap-web-client#338]
even when the option not to show it at startup is selected.
This is especially annoying on mobiles.
See https://lizmap.faunalia.eu/ for an example.

**Comment by mdouchin** [comment]
hum... I the source of Lizmap map web page, there is

```
      $( document ).ready( function() {
        lizMap.events.on({
          'uicreated':function(evt){
            $('#button-switcher').click();
          }
        });
      });
```

We must find why this is added in your example. Have you changed the source code ?

**Comment by nboisteault** [comment]
@pcav do you still have this issue ?

**Comment by nboisteault** [comment]
Fix by #1294
